app/components/Views/confirmations/components/UI/InfoRow/InfoRow.tsx

```diff
@@ -27,10 +27,12 @@ const InfoRow = ({
       style={{ ...styles.container, ...style }}
       testID={testID ?? 'info-row'}
     >
-      <View style={styles.labelContainer}>
-        <Text style={styles.label}>{label}</Text>
-        {tooltip && <Tooltip content={tooltip} />}
-      </View>
+      {label && (
```

Comment:
```suggestion
      {Boolean(label) && (
```

![CleanShot 2025-02-13 at 15 02 14@2x](https://github.com/user-attachments/assets/2f19936c-00c6-4d4b-a05a-fed8da97676f)